server/cluster/cluster.go

```diff
@@ -108,6 +108,7 @@ const (
 	// minSnapshotDurationSec is the minimum duration that a store can tolerate.
 	// It should enlarge the limiter if the snapshot's duration is less than this value.
 	minSnapshotDurationSec = 5
+	scanRegionLimit        = 1000
```

Comment:
How about making it a constant instead of being a parameter in `GetRegionSizeByRange`?